daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Visual.hs

```diff
@@ -82,20 +91,17 @@ moduleAndTemplates world mod = retTypess
         templates = NM.toList $ LF.moduleTemplates mod
         retTypess = map (\t-> (LF.tplTypeCon t, templatePossibleUpdates world t )) templates
 
-dalfsInDar :: Archive -> [BSL.ByteString]
-dalfsInDar dar = [fromEntry e | e <- zEntries dar, ".dalf" `isExtensionOf` eRelativePath e]
 
 dalfBytesToPakage :: BSL.ByteString -> (LF.PackageId, LF.Package)
 dalfBytesToPakage bytes = case Archive.decodeArchive $ BSL.toStrict bytes of
     Right a -> a
     Left err -> error (show err)
 
-darToWorld :: FilePath -> LF.Package -> IO LF.World
-darToWorld darFilePath pkg = do
-    bytes <- B.readFile darFilePath
-    let dalfs = dalfsInDar (toArchive $ BSL.fromStrict bytes)
-        pkgs = map dalfBytesToPakage dalfs
-    return (AST.initWorldSelf pkgs version1_4 pkg) 
+darToWorld :: ManifestData -> LF.Package -> LF.World
+darToWorld manifest pkg = AST.initWorldSelf pkgs version1_4 pkg
```

Comment:
Why not versionDev? I don't want another place to change on each upgrade

Reply:
The version parameter should have never been there in the first place https://github.com/digital-asset/daml/pull/1601 will remove it.